Docker/test/selenium/wdio.conf.js

```diff
@@ -49,19 +57,33 @@ exports.config = {
 	// ===================
 
 	// Level of verbosity: "trace", "debug", "info", "warn", "error", "silent"
-	logLevel: process.env.SELENIUM_LOG_LEVEL || 'warn',
+	logLevel: process.env.SELENIUM_LOG_LEVEL || 'error',
 
-	// Setting this enables automatic screenshots for when a browser command fails
-	// It is also used by afterTest for capturig failed assertions.
-	screenshotPath: process.env.LOG_DIR || __dirname + '/log',
+	// Setting this enables automatic screenshots for when a browser command fails assertions.
+	screenshotPath,
 
 	// Default timeout for each waitFor* command.
 	waitforTimeout: 30 * 1000,
 
 	// See also: http://webdriver.io/guide/testrunner/reporters.html
```

Comment:
This seems broken and also unnecessary? Not related to your changes, but maybe a chance to delete/update.

Reply:
Will catch in the code organisation branch.